app/javascript/app/components/emission-pathways-table/emission-pathways-table-selectors.js

```diff
@@ -71,7 +68,28 @@ export const filteredDataBySearch = createSelector(
   }
 );
 
+export const titleLinks = createSelector(
+  [getCategory, getData],
+  (categoryName, data) => {
+    if (!data || isEmpty(data) || !categoryName) return null;
+    const categoryId = {
+      Models: 'full_name',
+      Scenarios: 'name'
+    };
+    return data.map(d => ({
+      fieldName: categoryId[categoryName],
+      url: `${categoryName.toLowerCase()}/${d.id}`
+    }));
+  }
+);
+
+export const sortBy = createSelector([filteredDataBySearch], data => {
```

Comment:
Do we need this?? Not sure it is being used for anything, the table handles its own sorting now.